packages/ethereum-contracts/contracts/agreements/gdav1/SuperfluidPool.sol

```diff
@@ -214,6 +242,26 @@ contract SuperfluidPool is ISuperfluidPool, BeaconProxiable {
         return Value.unwrap(PDPoolMemberMU(pdPoolIndex, pdPoolMember).rtb(Time.wrap(time)));
     }
 
+    /// @inheritdoc ISuperfluidPool
+    function getMemberData(address memberAddr) external view override returns (MemberData memory memberData) {
+        return _membersData[memberAddr];
+    }
+
+    /// @inheritdoc ISuperfluidPool
+    function getTotalAmountReceivedByMember(address memberAddr) external view override returns (uint256) {
```

Comment:
add the assertion of `getTotalAmountReceivedByMember` usage in some of the existing test cases.